google/auth/external_account.py

```diff
@@ -413,3 +420,46 @@ def _initialize_impersonated_credentials(self):
             quota_project_id=self._quota_project_id,
             iam_endpoint_override=self._service_account_impersonation_url,
         )
+
+    @staticmethod
+    def validate_token_url(token_url):
+        _TOKEN_URL_PATTERNS = [
```

Comment:
not sure if this is should be lowercase since it is no longer global

Reply:
I'm considering it's a constant since we don't change our patterns.
Will let code owner to point out @arithmetic1728 @sai-sunder-s

Reply:
I think it is ok to keep it the current way